export/platform/gc_design_system/airtable/requirements.txt

```
@@ -0,0 +1,2 @@
+boto3==1.39.9
+requests==2.32.3
```

Could you bump this one patch version (these a vulnerability with this version)
```suggestion
requests==2.32.4
```